cfpb/owning-a-home
# Release journey

Reverses #618. Shows all links to the journey.

## Removals
- revert changes to `related_links.html` code so that so bottom nav appears
- add journey tests back to `navigation.feature`
- uncomment `process` links in `loan-estimate/loan-estimate-2.html`, `loan-estimate/loan-estimate-3.html`, and `mortgage-estimate/index.html`

## Testing

- check out branch locally
- run `sh frontendbuild.sh`
- run `grunt deploy`
- view front end and test that:
  - journey landing page is the homepage:
    - [http://localhost:7000/process/](http://localhost:7000/process)
    
  - the following pages have bottom nav with journey links:
    - [http://localhost:7000/explore-rates/](http://localhost:7000/explore-rates/)
    - [http://localhost:7000/loan-options/](http://localhost:7000/loan-options/)
    - [http://localhost:7000/loan-estimate/](http://localhost:7000/loan-estimate/)
    - [http://localhost:7000/closing-disclosure/](http://localhost:7000/closing-disclosure/)
    
  - the `Learn how to shop for multiple Loan Estimates` link to the journey is active on mortgage estimate page:
    - [http://localhost:7000/mortgage-estimate/](http://localhost:7000/mortgage-estimate/)

  - the journey links are active on [http://localhost:7000/loan-estimate/](http://localhost:7000/loan-estimate/)
    * [x] Check details, page 2, `Review the Services You can shop for`
    * [x] Check details, page 2, `Is the homeowner's insurance premium accurate`
    * [x] Check details, page 3, `Use the comparisons section...` (2 links)
  
- Confirm the follow features are *not* built:
  - [http://localhost:7000/loan-comparison/](http://localhost:7000/loan-comparison/)
  - [http://localhost:7000/prepare-worksheets/](http://localhost:7000/prepare-worksheets/)

- Check that smoke tests for journey links on the landing page are back 
  
## Notes
  - Still need to fix the `know before you owe` browser test on landing page

## Review

- @cfarm or @amymok 

## Checklist
